build.d/60-config-generate

```diff
@@ -1,9 +1,8 @@
 #!/bin/bash
-# Generate Odoo server configuration from templates
-set -e
-src="/opt/odoo/common/conf.d/* /opt/odoo/custom/conf.d/*"
-log INFO Merging $(ls $src | wc -l) configuration files in $OPENERP_SERVER
-conf=$(cat $src | envsubst)
-log DEBUG "Resulting configuration:
-$conf"
-echo "$conf" > $OPENERP_SERVER
+
+if [ "${CONFIG_BUILD}" != true ]; then
+    log WARNING Skipping Odoo config generation - will perform during entrypoint.
```

Comment:
If this is the new normal behavior, then the warning should go in the opposite case.